SetupDataPkg/Include/Library/PlatformConfigDataLib.h

```diff
@@ -16,6 +16,11 @@
 #include <Uefi.h>
 #include <ConfigStdStructDefs.h>
 
+#define SCHEMA_XML_HASH_VAR_NAME  L"SCHEMA_XML_HASH"
+#define SCHEMA_XML_HASH_GUID      {0x1321e012, 0x14c5, 0x42db, { 0x8c, 0xa9, 0xe7, 0x97, 0x1d, 0x88, 0x15, 0x18 }}
+#define SCHEMA_XML_HASH_LEN       33

```

Comment:
These definitions should live in a different header so that the platform does not have to include this in multiple places, potentially.